src/Sulu/Component/Content/Document/Behavior/AuditableBehavior.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Document\Behavior;
+
+use Sulu\Component\DocumentManager\Behavior\Audit\TimestampBehavior;
+
+/**
+ * This behavior combines the creator and changer with the created and changed dates.
+ */
+interface AuditableBehavior extends BlameBehavior, TimestampBehavior
```

Comment:
The `Behavior` suffix is a naming convention coming from the [Sulu Document Manager](https://github.com/sulu-io/sulu-document-manager). If we change that everybody should probably agree to the new convention (everywhere else we use `Interface` as a suffix, so I would prefer to name this interface `AuditableInterface`)